org.kde.kstars.json

```diff
@@ -7,15 +7,20 @@
     "rename-icon": "kstars",
     "finish-args": [
         "--share=ipc",
+        "--share=network",
+        "--device=all",
         "--socket=fallback-x11",
         "--socket=wayland",
         "--socket=pulseaudio",
-        "--share=network",
-        "--device=all",
         "--system-talk-name=org.bluez",
+        "--system-talk-name=org.freedesktop.UDisks2",
         "--env=GSCDAT=/app/share/GSC",
-        "--filesystem=home",
-        "--system-talk-name=org.freedesktop.UDisks2"
+        "--filesystem=/usr/share/astrometry:ro",
+        "--filesystem=/usr/share/hips:ro",
```

Comment:
`/usr` is reserved path it can't be used in `--filesystem`

Reply:
We can't package those. Astrometry files can be up to 30GB in size. Similarly, hips can be up to 100GB in size. Usually users already have these installed independent of KStars. They're just data files, nothing else.

Reply:
the problem is that 1. they can't be accessed like that 2. if they are need for the app then they need to come that way. There is no way to have them being installed automatically where you are trying to locate them. 3. And please test your stuff before claiming it's necessary to solve an issue.

Reply:
So we have two methods:
1. host access: Simple, but not sandboxy.
2. Ask users to pick the directories using document. Not convenient but can work.

Reply:
If paths for those folders are selectable somewhere then access to them can work through portals. If they are hardcoded to `/usr/share` then I don't see solution since` /usr/share` always comes from flatpak runtime and it isn't overridable.

Reply:
Nothing folders are hard-coded in KStars, all are selectable. I tested and it works, but it would require extra steps for new users to do.

Reply:
yeah, there is no alternative though, even `host` permission would put them into different path than `/usr/share` which would need to be selected by users